hugegraph-core/src/main/java/com/baidu/hugegraph/auth/StandardAuthManager.java

```diff
@@ -472,7 +475,7 @@ public UserWithRole validateUser(String username, String password) {
 
     @Override
     public UserWithRole validateUser(String token) {
-        String username = (String) this.tokenCache.get(IdGenerator.of(token));
+        String username = this.tokenCache.get(IdGenerator.of(token));
```

Comment:
replace CACHE_EXPIRE with tokenCache.expire() at line 493.
and improve the way to invalid pwd and token cache.